src/test/java/com/chessmates/lichess/data/LichessDataServiceImplTest.groovy

```diff
@@ -217,11 +224,7 @@ class LichessDataServiceImplTest extends Specification {
         service.updatePlayers()
 
         then:
-        1 * playerRepository.save({ it.id == 'jfaker' })
-        1 * playerRepository.save({ it.id == 'riciardos' })
-        1 * playerRepository.save({ it.id == 'samei07' })
-        1 * playerRepository.save({ it.id == 'sydeman' })
-        1 * playerRepository.save({ it.id == 'torrlane' })
+        1 * playerRepository.saveAll(_)
```

Comment:
Do we still want to test that the thing we are saving is what we expect?
```
1 * playerRepository.saveAll({
    it[0].id == 'jfaker' &&
    ... // so forth
})
```

There's probably a nicer way than what I've done above.